### ISSUE TITLE
Create a .gif of an example

### ISSUE THREAD

**Issue by MikeBruns** [MikeBruns/expect#11]
Record going through an example of ways to use the package and make it a gif that can be included in the README

**Comment by OmarAguinaga** [comment]
hey just did a PR on this issue. Let me know if thats what you were looking for. Thanks.

**Comment by MikeBruns** [comment]
@OmarAguinaga That looks fantastic! 
This will definitely help users understand what they are installing, very helpful.

**Comment by MikeBruns** [comment]
@OmarAguinaga one thing I saw that atom does not like, which I would have never guessed, was that it dose not like the direct path to a file in the project. I had to switch it to (https://raw.githubusercontent.com/MikeBruns/expect/master/example.gif). It would work on the local README and the atom page https://atom.io/packages/expect but not in the atom application.

Which I am not particularly keen on this direct path solution, but is the only thing that works.
https://discuss.atom.io/t/my-images-dont-show-at-atom-io-packages/11629 

I love the way it looks :heart_eyes:
